src/factories/TransferProxyFactory.sol

```diff
@@ -44,6 +44,8 @@ contract TransferProxyFactory is ITransferProxyFactory {
 
         address proxy = address(new TransferProxy(poolManager, destination));
         proxies[destination] = proxy;
+
+        emit DeployTransferProxy(destination, proxy);
```

Comment:
This (and the other event changes) are unrelated to the PR, just a small change I wanted to do that didn't seem worth a separate PR.